src/components/SingleRev.jsx

```diff
@@ -46,7 +51,7 @@ export const SingleRev = () => {
           return comment;
         });
       });
-  }, [reviewId]);
+  }, [reviewId, voting]);
```

Comment:
Do we want voting in the dependancy array? This should be sorted by optimistic rendering and the api call.